website/content/guide/request.md

```diff
@@ -174,7 +174,7 @@ type (
 	}
 
 	CustomValidator struct {
-		validator *validator.Validate
+		Validator *validator.Validate
```

Comment:
Why is this needed to be made public? It is not required outside of the CustomValidator.